server/src/test/resources/reference.conf

```diff
@@ -1 +1,5 @@
-working-directory = "/tmp"
\ No newline at end of file
+object-service {
```

Comment:
I don't think these are needed...we need to test configs are reading environment variables properly too...